domain/consensus/processes/coinbasemanager/coinbasemanager_test.go

```diff
@@ -78,3 +79,50 @@ func TestCalcDeflationaryPeriodBlockSubsidy(t *testing.T) {
 		}
 	}
 }
+
+func TestBuildSubsidyTable(t *testing.T) {
```

Comment:
Add t.Skip() so this test won't run every time the test suite is run

Reply:
But I want it to run in case base subsidy is modified and this method is not modified too

Reply:
Ok, fixed it so now table printing is using the test logger and not console print